MeetU/MeetU/Scripts/Views/Meetup/Index.html

```diff
@@ -103,9 +103,9 @@
                         </div>
                     </div>
                 </div>
-                <div class="meetup-functional-button-wrapper" ng-show="meetupView.meetup.sponsor === userId">
-                    <a class="glyphicon glyphicon-edit meetup-functional-button" href="#/Meetup/Edit/{{meetupView.meetup.id}}"></a>
-                    <span class="glyphicon glyphicon-trash meetup-functional-button" ng-click="deleteMeetup(meetupView.meetup.id)"></span>
+                <div class="meetup-functional-button-wrapper" ng-show="meetupView.meetup.sponsor === indexVm.userId">
+                    <a class="glyphicon glyphicon-edit meetup-functional-button" href="#/Meetup/Edit/{{ meetupView.meetup.id }}"></a>
+                    <span class="glyphicon glyphicon-trash meetup-functional-button" ng-click="indexVm.deleteMeetup(meetupView.meetup.id)"></span>
                 </div>
             </div>
             <hr class="meetup-separator" />
```

Comment:
in loading-circle tag and the following div, some variables might need to be modified as well.